code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -11,7 +11,7 @@
 	var/min_health = -100
 	var/cleaning = 0
 	var/patient_laststat = null
-	var/list/injection_chems = list("inaprovaline", "dexalin", "bicaridine", "kelotane","anti_toxin", "alkysine", "imidazoline", "spaceacillin", "paracetamol") //The borg is able to heal every damage type. As a nerf, they use 750 charge per injection.
+	var/list/injection_chems = list("inaprovaline", "dexalin", "bicaridine", "kelotane", "anti_toxin", "spaceacillin", "paracetamol") //The borg is able to heal every damage type. As a nerf, they use 750 charge per injection.
```

Comment:
So wouldn't this undo the medihound buff/nerf thing? They still cost 750 per injection, but can now no longer heal every damage type?

Reply:
I don't really have any evidence that wasn't referring to the 4 main types. 'eye damage' isn't what I'd call in 'every damage type', and they can't heal genetic damage so why would that comment exist if that's what it meant?

That's how I interpret it anyway.

Reply:
Reduced cost somewhat to compensate. It's still more than the borg hypo because unlike the hypo, it has an infinite supply whereas the hypo takes time to regenerate the stockpile (several minutes).